module.json

```diff
@@ -546,7 +546,7 @@
             "None": "none",
             "Orange-400": "#ffcc78",
             "Orange-600": "#FFab33",
-            "Green-400": "77c785",
+            "Green-400": "#77c785",
```

Comment:
good catch

Reply:
Lucky, the only thing I tried after implementation that didn't work :)